### ISSUE TITLE
Render vlc to Tk photo images

### ISSUE THREAD

**Issue by undroidwish** [ray2501/tkvlc#1]
Danilo,

few days ago, I've found your tkvlc project and couldn't resist to improve it somewhat in order to be able to use it in undroidwish. This doesn't have directly usable platform handles (HWND or Window IDs) as in plain Win32 or X1. Thus I introduced rendering to photo images among other minor improvements, e.g. dynamic linking of libvlc.so. If you find it in parts usable, I'll be glad if you take over some of my changes.

See this AndroWish check-in for more information:
    https://www.androwish.org/index.html/info/f971f95c80fc9ad7

All the best,
Christian

**Comment by undroidwish** [comment]
Sorry, wrong check-in given, the correct one is
https://www.androwish.org/index.html/info/0c3d802cda75011a

**Comment by undroidwish** [comment]
I've added few other features up to check-in https://www.androwish.org/index.html/info/59a46558bf15bfd0

* event callback
* introspection
* repeated playback

The event callback is esp. useful for newly rendered photo images to be used as a live texture in a Canvas3D widget. A demo is included, see the examples directory.

I've tried it out on Raspberry Pi (Debian 9) and Odroid C2 (Ubuntu 18.04): on the Raspi, rendering to photo images doesn't work at all, whereas on the Odroid the libvlc seems to be performing very well.
